src/melange-playground/Makefile

```diff
@@ -0,0 +1,2 @@
+serve:
+	chapter=melange-playground make -C ../.. serve
```

Comment:
~Are the Makefiles in these folders useful anymore? I thought one would just call `npm run docs:dev` (or `bun run docs:dev`) and have access to all chapters at once.~

Edit: oh, I see, these are to run the actual code of the book. And the `chapter` is read from top level command.